src/renderer/Presentational/ModalManager/modalUtils.tsx

```diff
@@ -0,0 +1,60 @@
+import { ThemeSetting } from 'main/state/settings';
+import React from 'react';
+import Node from 'common/node';
+
+export interface ModalConfig {
+  executionClient?: string;
+  consensusClient?: string;
+  storageLocation?: string;
+  theme?: ThemeSetting;
+  isOpenOnStartup?: boolean;
+  selectedNode?: Node;
+  isDeleteStorage?: boolean;
+  settingsConfig?: object;
+  newDataDir?: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
+}
+
+export const modalRoutes = Object.freeze({
+  addNode: 'addNode',
+  nodeSettings: 'nodeSettings',
+  preferences: 'preferences',
+  addValidator: 'addValidator',
+  clientVersions: 'clientVersions',
+  stopNode: 'stopNode',
+  removeNode: 'removeNode',
+  updateUnavailable: 'updateUnavailable',
+});
+
+export const modalOnChangeConfig = async (
```

Comment:
Could you add a function description comment for this? It seems key to adding a new modal. Let's say a description geared for me, if I wanted to add a new modal